dev.Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN mkdir /code/requirements/
 COPY requirements/dev.txt /code/requirements/
 RUN pip install -r requirements/dev.txt --compile
 
+COPY posthog.json /code/
```

Comment:
Trying to build locally failed because this file didn't exist

Reply:
Hum, that didn't make sense indeed. Just a few lines later with `COPY . /code/` it gets copied anyway. Removed.